src/test/java/com/example/survey/models/RangeQuestionTest.java

```diff
@@ -0,0 +1,23 @@
+package com.example.survey.models;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RangeQuestionTest {
+
+    @Test
+    void getQuestion() {
+        String expected =  "how much does kirin bench (in lsb)?";
+        RangeQuestion rq = new RangeQuestion(expected, 45, 240);
+        assertTrue(expected.equals(rq.getQuestion()));
+    }
+
+    @Test
+    void testToString() {
+        RangeQuestion rq = new RangeQuestion("how much does kirin bench (in lsb)?", 45, 240);
```

Comment:
```suggestion
        RangeQuestion rq = new RangeQuestion("how much does kirin bench (in lbs)?", 45, 205);
```

Only counting paused bench from now on 🙃